### DISCUSSION TITLE
Activate / deactivate camera siren through HA automation?

### DISCUSSION THREAD

**Discussion by V4ler1an** [fuatakgun/eufy_security#1096] [Q&A]
I have S100 wall light cameras. When using the Eufy mobile app there is an option to manually trigger the camera's siren when viewing the live feed. I can't seem to find any way to do the same through the Home Assistant integration. There are buttons like 'button.trigger_alarm' and 'button.reset_alarm' but these don't sound the camera siren. I'm assuming they are for triggering the HomeBase alarm system which is not what i'm looking for. My goal is to setup a toggle in HA dashboard which just turns the siren on / off. Can anyone help me with this?

Update: Turns out the feature is available but doesn't currently work. Error below:

Automation alarm trigger step:
![image](https://github.com/fuatakgun/eufy_security/assets/134781183/f7d2b0f0-0775-4e87-a88b-442a3f5274f5)

Error code - 'device_not_supported':
![image](https://github.com/fuatakgun/eufy_security/assets/134781183/297e7191-ddc9-4e16-b75a-64d00df66120)

**Comment by AwesomeNinou** [comment]
There is a second trigger_alarm entity that does trigger the siren on my floodcam but only for few seconds…

**Reply by V4ler1an** [reply]
For my S100 wall lights at least, the feature looks to be broken. I've updated the post with the error.
